app/assets/javascripts/discourse/app/raw-templates/flat-button.hbr

```diff
@@ -1,3 +1,3 @@
-<button class='btn-transparent {{class}}' title='{{i18n "topics.bulk.toggle"}}'>
+<button class='btn-transparent {{class}}' title='{{i18n title}}'>
```

Comment:
This was just a minor refactoring to get the title dynamically instead of always getting the hardcoded for topics.